packages/localize/tsconfig.json

```diff
@@ -1,20 +1,22 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "module": "commonjs",
+    "target": "es2018",
```

Comment:
why the target change?

Reply:
Not really changed, this tsconfig used to be for the CLI, but now it's for the client. It's actually the same as what was `tsconfig.client.json`